client/src/components/FormBuilderModal/FormBuilderModal.js

```diff
@@ -16,14 +18,17 @@ class FormBuilderModal extends SilverStripeComponent {
   }
 
   handleLoadingError(schema) {
-    if (this.props.showErrorMessage) {
+    const providesOnLoadingError =
+      typeof this.props.onLoadingError === 'function'
+      && this.props.onLoadingError !== noop;
```

Comment:
Can simplify to `this.props.onLoadingError !== noop`, we have the propTypes validation which checks for a function type.
Users shouldn't be passing in `null` to the prop anyway.

Reply:
Updated.